data/json/effects_on_condition/npc_eocs/hallucination_eocs.json

```
@@ -0,0 +1,195 @@
+[
+  {
+    "type": "effect_on_condition",
+    "id": "hallucinate_effect",
+    "global": false,
+    "recurrence": [ "2 hours", "6 hours" ],
+    "condition": {
+      "and": [
+        { "not": { "u_has_effect": "sleep" } },
+        { "not": { "u_has_effect": "took_thorazine" } },
+        { "u_has_trait": "SCHIZOPHRENIC" }
```

```suggestion
        { "u_has_trait": "SCHIZOPHRENIC" }
```
Should this be an `"or": " "u_has_effect": "hallu"` ?

---

Yes, it should. I'll fix that.